plugins/action/synchronize.py

```diff
@@ -389,7 +389,7 @@ def run(self, tmp=None, task_vars=None):
 
         # If launching synchronize against docker container
         # use rsync_opts to support container to override rsh options
-        if self._remote_transport in ['docker', 'community.general.docker', 'buildah', 'containers.podman.buildah'] and not use_delegate:
+        if self._remote_transport in ['docker', 'community.general.docker', 'community.docker.docker', 'buildah', 'containers.podman.buildah'] and not use_delegate:
```

Comment:
This line is too long according to Zuul:
```
2021-03-02 15:56:10.113840 | centos-8 | ERROR: plugins/action/synchronize.py:392:161: E501: line too long (164 > 160 characters)
```
(https://e95bd11f717195a90d9d-c7dcb72ad68df77a0c76e81c84cbc136.ssl.cf2.rackcdn.com/144/a5e46017e973c4c91dfec1b0414e20335c677d0e/check/ansible-test-sanity-docker-stable-2.10/cc43b2f/job-output.html#l9902)